core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -521,7 +520,7 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     sendRecords(numRecords, topicPartition)
   }
 
-  @Test(expected = classOf[GroupAuthorizationException])
+  @Test(expected = classOf[TopicAuthorizationException])
   def testConsumeWithNoAccess(): Unit = {
```

Comment:
Maybe we could mention in the name that this test case is using manual assignment?